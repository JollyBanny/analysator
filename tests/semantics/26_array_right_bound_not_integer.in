var a: string;
    c: array [1..a] of integer;

begin
    write(a);
end.