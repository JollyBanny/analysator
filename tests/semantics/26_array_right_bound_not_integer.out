[Line: 2; Column: 16]	index '..' is not integer
